src/ProjectNeon/Assets/Features/Combatants/InMemoryStats.cs

```diff
@@ -9,5 +11,7 @@ public sealed class InMemoryStats : IStats
         public float Armor { get; set; }
         public float Resistance { get; set; }
         public IResourceType[] ResourceTypes { get; set; } = new IResourceType[0];
+        public Func<int, bool> active;
```

Comment:
Should be a Public Property. 

Must be capitalized.

Must be initialized with a default, to prevent `NullRefs`